### ISSUE TITLE
Update staging instance with bug fixes

### ISSUE THREAD

**Issue by maccraymer** [ISO-TC211/iso-geodetic-registry#80]
Please update the staging instance of the Registry with Florian’s bug fixes so I can test that all the issues have been resolved. Please do **not** update the public instance until I have finished testing the staging instance.

The ISO Control Body would appreciate if this could be given the **highest priority** right now as there are errors in the current Registry content that we are unable to correct until the bug fixes are tested and implemented. As it presently exists, the Registry is unusable without these bug fixes and the Excel importing function, and, worse, it continues to provide erroneous information to the public. Thanks!

**Comment by ronaldtse** [comment]
@maccraymer sorry we had to take the time to finish testing Florian's bug fixes with the existing database in order to deploy. Most issues are now verified.

@phuonghuynh will be deploying staging once we are able to do so.

**Comment by phuonghuynh** [comment]
Staging updated to "master" branch now, which contains fixes from florianesser

**Comment by maccraymer** [comment]
Thanks Ron & Phuong! Can this issue be closed? I probably should have just emailed you instead of making an “issue” out of this request.

**Comment by maccraymer** [comment]
@ronaldtse & @phuonghuynh, can you please update both the public & staging instances of the registry with the latest master that includes all closed bug fixes and enhancements. The currently open issues are not as urgent and should not preclude the CB from starting to update the Registry content after getting an answer to my last comment in #92.

And should we keep this issue open for me to periodically request updating the staging and public instances? Rather than creating a new issue each time.

**Comment by phuonghuynh** [comment]
@maccraymer staging has been updated with master, could you help to check it before I deploy master to public server?

**Comment by phuonghuynh** [comment]
@maccraymer public site has been deployed

**Comment by maccraymer** [comment]
@phuonghuynh I checked the staging instance and found another 500 error using the Details action after the RM accepts a clarification proposal (see #95).

**Comment by ronaldtse** [comment]
This should have been resolved.

**Comment by phuonghuynh** [comment]
@ronaldtse this issue should not be close to let maccraymer periodically request updates for servers.

>And should we keep this issue open for me to periodically request updating the staging and public instances? Rather than creating a new issue each time.

**Comment by maccraymer** [comment]
Please update both the public and staging instances with the latest master. The open bug issues are not critical for revising the existing content. Thanks!

**Comment by maccraymer** [comment]
Just wondering when you will be able to update both the public and staging instances. A couple of CB members want to correct some of their current content before the end of the week. Thanks.

**Comment by ronaldtse** [comment]
@maccraymer we will be updating this today. Thanks!

**Comment by phuonghuynh** [comment]
Updated both staging+production to master githash `67180d4`

**Comment by maccraymer** [comment]
Please update the public registry to the same version as the staging instance. The staging instance appears to be fixed for GML output but the public registry is still giving the 500 error.

**Comment by phuonghuynh** [comment]
Yes, i will revert changes made for Java8 to Java7, this will fix Linux server GML view

**Comment by maccraymer** [comment]
Thanks @phuong! Maybe Java7 will also fix the new issue #112 I just reported (500 error when deleting a user). I only get the error with the public instance, not the staging one. Let me know here when you have reverted and I will check if the delete error is still present. Thanks.

**Comment by maccraymer** [comment]
The staging registry appears to be corrupted after much testing of different kinds. Can you please reinstall the database. Please install the current public database for the staging registry so we have an up to date version. But please keep a backup of the older versions in case we need to revert.

The public registry may also be corrupted (at least the one item 138). What backups are available for the public registry's database? Can we step back in time and check older versions of the database in the staging registry to see when the item might have become corrupted or if was corrupted all a long?

Thanks!

**Comment by phuonghuynh** [comment]
@ronaldtse should we implement cloudwatch for this DB backup action?

**Comment by ronaldtse** [comment]
@phuonghuynh yes, let's:

1. create a cloudwatch periodic job to perform the backup from production (we already have this, right)?
2. update the staging site to restore the production data using cloudwatch periodically (every sunday?)
